src/plugins/lua/luaplugin.cpp

```diff
@@ -697,4 +763,11 @@ void LuaPlugin::writeTextProperties(LuaTableWriter &writer, const MapObject *map
     }
 }
 
+void LuaUtilWriter::mapDir(const QString &dir)
```

Comment:
Naming style: should be called `setMapDir`.

Reply:
Done